pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -55,6 +55,7 @@
 import org.apache.pulsar.broker.loadbalance.LeaderElectionService;
 import org.apache.pulsar.broker.loadbalance.LoadManager;
 import org.apache.pulsar.broker.loadbalance.ResourceUnit;
+import org.apache.pulsar.broker.loadbalance.extensions.ExtensibleLoadManagerImpl;
```

Comment:
It would be great to add unit tests in NamespaceServiceTest for these if-else variations. (assert if the new/old load manager is called for these Namespace public funcs).

Or, please add a TODO comment if we want to add such tests later.
```
if (ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(config)) {
// new logic
} else {
// old logic
}
```

Reply:
Good point. I'll add it later.

Reply:
After thinking about it, I guess adding tests later might be better since some methods of `NamespaceService` still need to add new logic when using extensible load manager.

Reply:
nit: Actually, we can have a separate PR to integrate ExtensibleLoadManagerImpl into the existing code(to the NamespaceService). This PR could be ExtensibleLoadManagerImply only.

Reply:
I tried to split it into two PRs, but `ExtensibleLoadManagerImpl ` test dependency topic lookup because we are using `TableView` in `ServiceUnitStateChannel`, so I have to implement the load manager function.